pkg/gatewayserver/scheduling/scheduler.go

```diff
@@ -447,3 +448,20 @@ func (s *Scheduler) TimeFromTimestampTime(t uint32) (ConcentratorTime, bool) {
 	}
 	return s.clock.FromTimestampTime(t), true
 }
+
+// SubBandStatistics returns a map with the usage statistics of each sub band.
+func (s *Scheduler) SubBandStatistics() map[string]*ttnpb.GatewayConnectionStats_SubBandStatistics {
+	res := make(map[string]*ttnpb.GatewayConnectionStats_SubBandStatistics)
+
+	for _, sb := range s.subBands {
+		if sb == nil {
+			continue
+		}
+		key := fmt.Sprintf("%.1f-%.1f", float32(sb.MinFrequency)/1000000, float32(sb.MaxFrequency)/1000000)
```

Comment:
This is too much representation in the API; I'd keep this in a list along with the frequencies, and let the client do the sorting and display